upload.sh

```diff
@@ -60,7 +63,10 @@ upload_package() {
 
     # Upgrade pip.
     python -m pip install --quiet --upgrade pip
-
+    
+    # Upgrade build.
+    python -m pip install --quiet --upgrade build
```

Comment:
isn't build getting used in `build_package`?  should we upgrade earlier?

Reply:
We were creating it but only before the twine install, not for the build.

Reply:
doing it all in a venv is fine with me